Godot.gitignore

```diff
@@ -3,6 +3,8 @@
 .import/
 export.cfg
 export_presets.cfg
+# Imported translations (automatically generated from CSV files)
```

Comment:
It looks like this can be it's own section, so should we add a newline before this comment header?

```suggestion

# Imported translations (automatically generated from CSV files)
```